src/components/Collapse/Collapse.vue

```diff
@@ -3,7 +3,6 @@
         class="d-inline-block"
         data-bs-toggle="collapse"
         :data-bs-target="`#${id}`"
-        @click="emit('click:toggle')"
```

Comment:
So how can I now listen to a click on toggle?

Reply:
This can't be done by the `@click="$attrs['onClick:toggle']"` method because then Vue gives a warning that the toggle emit is not defined. Defining the toggle event in the emits gives the same warning shared before, that all other emits in `CollapseEvents` are not defined